community/examples/ramble.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+
+blueprint_name: ramble
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: ramble
+  region: us-central1
+  zone: us-central1-c
+
+deployment_groups:
+- group: primary
+  modules:
+  - id: network1
+    source: modules/network/pre-existing-vpc
+
+  - id: spack
+    source: community/modules/scripts/spack-install
+    settings:
+      install_dir: /spack
+
+  - id: ramble-setup
+    source: community/modules/scripts/ramble-setup
+    settings:
```

Comment:
Shouldn't ramble be using spack?

Reply:
Not this module. I don't need ramble to use spack until the ramble-execute module.